### ISSUE TITLE
Pattern: Card - Accessibility - When focus reaches Click here link, Voiceover announces "table start landmark" which confuses the user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1565]
## Detailed description

Issue :- When focus reaches this click here link, Voiceover announces "table start landmark" which confuses the user.
Expected - Since there is no landmark or table found here, Screen reader should not be announcing this.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/10e2a809-634d-4eca-89c4-6f1968199d96)

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 
Some updates were done to the cards recently,
can you please test it again? thanks!

**Comment by Praveen-sr** [comment]
@RaphaelAmadeu Ram re checked, the following issue is not there now. 
Instead Voice-over iPhone is announcing as "footer region" when focus comes to the click here link. 
He is suggesting - If this link is not within a footer section, the footer landmark should be removed from the code.
